src/index.css

```
@@ -1,6 +1,7 @@
 :root {
   --color--gray--darker: #343434;
   --color--gray--dark: #444;
+  --color--gray--medium-dark: #999;
```

I found this a little hard to read, and looked up what the [accessibility standards for color contrast](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast) are. In short, the minimum is 4.5:1, and ideal is 7:1; [#999 against white](https://webaim.org/resources/contrastchecker/?fcolor=999999&bcolor=FFFFFF) has a ratio of 2.84:1.

`#757575` has a ratio of 4.6:1, which clears the minimum; is that subtle enough?

```suggestion
  --color--gray--medium-dark: #757575;
```

---

Great note, will change!